src/app/hero/home/home-container/home-container.component.html

```diff
@@ -1,2 +1,19 @@
 <mu-header></mu-header>
+<div class="content">
+  <div class="funtionalities">
+    <div class="search-bar">
+      <input type="text" placeholder="  Search... " #searchedString>
+      <button type="button" (click)="searchHeroes(searchedString.value)">GO</button>
```

Comment:
using and id and a value is also fine, or you can use form controls.. this s fine for now